Documentation/user-documentation/user-documentation.md

```diff
@@ -0,0 +1,115 @@
+- The frontend is only to be used in landscape (width > height).
+- When using it on small screens, the bottom control panel is hidden and can be opened by tapping the caret at the
+  bottom.
+- There's a light and a dark mode, based on your OS color settings.
+
+![frontend-full.png](frontend-full.png)
+
+<div style="page-break-after: always;"></div>
+
+### Top control panel
+
+In the top left corner, you'll find a control panel for general control of the oscilloscope.
+
+![frontend-top-control-panel.png](frontend-top-control-panel.png)
+
+#### Controls (from left to right)
+
+- **Start/Stop:** Toggle processing all incoming samples.
+- **Reset:** Reset the indicators and the oscilloscop to the initial state and stop processing incoming samples.
+- **Ground:** While pressed, set all rendered lines to zero.
+- **Distribute vertically:** Distribute all enabled lines vertically to take up the whole oscilloscope space.
+- **Settings:** Open the settings modal for saving and loading presets.
+
+<div style="page-break-after: always;"></div>
+
+### Oscilloscope
+
+![frontend-oscilloscope.png](frontend-oscilloscope.png)
+
+#### Components:
+
+- **Line indicators:** Show the min and max values of all enabled channels with respect to their indidividual offsets
+  and amplitudes.
+- **Plot:** Renders the lines of enabled channels with respect to their individual offsets, amplitudes, sweep.
+  speeds and thickness
+- **Text indicators:** Show a textual min max value representation of all enabled channels.
+
+<div style="page-break-after: always;"></div>
+
+### Bottom control panel
+
+![frontend-bottom-control-panel.png](frontend-bottom-control-panel.png)
+
+The bottom control panel provides channel-based settings that influence the representation of incoming samples.
+
+#### Controls
+
+- **Start/Stop:**
+    - Toggle processing of a channel's samples.
+    - Affects the plot and the indicators.
+- **Thickness:**
+    - Toggle thick rendering of a channel's plot line.
+    - Affects the plot.
+- **Offset:**
+    - Set the vertical offset of a channel's plot line.
+    - Values range from -1 to 1 and are related to the plot's vertical height. A value of 1 means that the zero line of
+      a channel is located at the top end of the plot.
+    - Affects the plot and the line indicators.
+- **Time sweep:**
+    - Set the speed in which a channel's line crosses one horizontal division of the plot.
+    - Affects the plot.
+- **Amplitude:**
+    - Set the value that's been multiplied with a channel's plot line.
+    - Values range from 0.2 to Infinity and represent the reciprocal value of the vertical scaling. A reciprocal value
+      of 0.2 (0.2 = 1/5) means that the channel's values show as multiplied by 5.
+    - Affects the plot and the line indicators.
+
+<div style="page-break-after: always;"></div>
+
+### Settings
+
+![frontend-settings.png](frontend-settings.png)
+
+The settings modal provides a form to save and load the current settings.
+
```

Comment:
Maybe it would be useful to provide a little more information. For example, which settings are saved exactly and will old ones be overwritten? Do they need to be backed up before saving the new ones?

Reply:
That's a good point. IMO a simple list like
- Offset
- Ampltiude
- [rest of attributes]
would suffice.